package.json

```diff
@@ -59,72 +60,66 @@
   },
   "dependencies": {
     "@agm/core": "^1.1.0",
-    "@angular/animations": "^9.1.12",
-    "@angular/cdk": "^9.2.4",
-    "@angular/common": "^9.1.12",
-    "@angular/compiler": "^9.1.12",
-    "@angular/core": "^9.1.12",
-    "@angular/forms": "^9.1.12",
-    "@angular/localize": "^9.1.12",
-    "@angular/material": "^9.2.4",
-    "@angular/material-moment-adapter": "^9.2.4",
-    "@angular/platform-browser": "^9.1.12",
-    "@angular/platform-browser-dynamic": "^9.1.12",
-    "@angular/platform-server": "^9.1.12",
-    "@angular/router": "^9.1.12",
-    "@auth0/angular-jwt": "^4.2.0",
-    "@mat-datetimepicker/core": "^4.1.0",
-    "@mat-datetimepicker/moment": "^4.1.0",
-    "@ng-bootstrap/ng-bootstrap": "^6.2.0",
-    "@ngx-translate/core": "^12.1.2",
-    "@ngx-translate/http-loader": "^4.0.0",
+    "@angular/animations": "~11.0.0",
+    "@angular/cdk": "^11.0.0",
+    "@angular/common": "~11.0.0",
+    "@angular/compiler": "~11.0.0",
+    "@angular/core": "~11.0.0",
+    "@angular/forms": "~11.0.0",
+    "@angular/material": "^11.0.0",
+    "@angular/material-moment-adapter": "^11.0.0",
+    "@angular/platform-browser": "~11.0.0",
+    "@angular/platform-browser-dynamic": "~11.0.0",
+    "@angular/router": "~11.0.0",
+    "@auth0/angular-jwt": "^5.0.1",
+    "@mat-datetimepicker/core": "^5.1.0",
+    "@mat-datetimepicker/moment": "^5.1.0",
+    "@ng-bootstrap/ng-bootstrap": "^8.0.0",
+    "@ngx-translate/core": "^13.0.0",
+    "@ngx-translate/http-loader": "^6.0.0",
     "angular-moment-picker": "^0.10.2",
     "angular2-chartjs": "^0.5.1",
     "bootstrap": "^4.5.3",
     "bootstrap-material-design": "^4.1.3",
     "bootstrap-notify": "^3.1.3",
     "bootstrap-select": "^1.13.18",
     "bootstrap-tagsinput": "^0.7.1",
-    "canvas-gauges": "^2.1.7",
     "chart.js": "^2.9.4",
     "chartjs-plugin-datalabels": "^0.7.0",
     "chartjs-plugin-zoom": "^0.7.7",
     "file-saver": "^2.0.2",
+    "hammerjs": "^2.0.8",
     "http-status-codes": "^2.1.4",
     "jasny-bootstrap": "^4.0.0",
     "jquery": "^3.5.1",
     "moment": "^2.29.1",
     "moment-duration-format": "^2.3.2",
     "moment-timezone": "^0.5.31",
     "ngx-captcha": "^8.0.1",
-    "ngx-daterangepicker-material": "^3.0.4",
+    "ngx-daterangepicker-material": "^4.0.1",
     "ngx-google-places-autocomplete": "^2.0.5",
-    "ngx-moment": "^3.5.0",
-    "perfect-scrollbar": "^1.5.0",
+    "ngx-moment": "^5.0.0",
     "popper.js": "^1.16.1",
-    "rxjs": "^6.6.3",
-    "socket.io-client": "^2.3.1",
```

Comment:
"socket.io-client": "^2.3.1",
Do not remove that dep ;)

Reply:
As it's working without I believed this was not necessary, as well as for rxjs...

Reply:
Done

Reply:
It probably have added as a sub-dependency of a dependency. But you then do not control the version used and it's important for socket.io.